package.json

```diff
@@ -60,6 +60,7 @@
     "react-spinkit": "^1.1.11",
     "react-translate-component": "^0.11.1",
     "redux": "^3.6.0",
+    "redux-thunk": "^2.1.0",
```

Comment:
no idea actually. That stuff has an extremly odd, unpredictable behavior. Will need to check it out again before merging